plugin/file/wildcard_test.go

```diff
@@ -296,3 +366,13 @@ example.org.		IN	NS	b.iana-servers.net.
 *.intern.example.org.   IN      A       127.0.1.52
 foo.example.org.        IN      A       127.0.0.54
 `
+
+const cornerCasesWildcard = `; example.org test file with wildcard corner cases
+$TTL 3600
+example.org.		IN	SOA	sns.dns.icann.org. noc.dns.icann.org. 2015082541 7200 3600 1209600 3600
+example.org.		IN	NS	b.iana-servers.net.
+*.d.example.org.        IN      A       127.0.1.53
+z.+.d.example.org.		IN		A		127.0.1.53
+x.&.d.example.org.		IN		A		127.0.1.53
```

Comment:
These IPs should be different from the wild card entry IP - So that an answer from these records can be discerned from an (incorrect) answer synthesized from the wildcard record.

Reply:
@chrisohaver Change made.